lib/Config.js

```diff
@@ -469,7 +470,13 @@ class Config {
             if (process.env.S3AUTH_CONFIG) {
                 authfile = process.env.S3AUTH_CONFIG;
             }
-            const authData = require(authfile);
+            let authData = require(authfile);
```

Comment:
If `authfile` is invalid JSON, this would throw an error. Maybe it's okay, but it could be helpful to check that first.